src/server/identity/server/dex_web.go

```diff
@@ -23,41 +24,35 @@ var webDir = "/dex-assets"
 type dexWeb struct {
 	sync.RWMutex
 
-	// `issuer` must be a well-known URL where all pachds can reach this server.
-	// Dex usually loads it from a config file, but it can be clumsy to find the
-	// exact right value. Instead of requiring the user to change the config
-	// we support updating it via an RPC.
```

Comment:
Any chance you'd be OK preserving this comment in `src/server/identity/db.go`? (to document what's stored in the config table, basically)